go_utils/photo_download.py

```diff
@@ -276,6 +351,11 @@ def get_lc_download_targets(
         The column name of the column that contains the east photo urls. If not specified, these photos will not be included.
     west_photo : str, default = "lc_WestPhotoUrl"
         The column name of the column that contains the west photo urls. If not specified, these photos will not be included.
+    include_in_name : list of str, default=[]
+        A list of coloumn names to include into the downloaded photo names. The order of items in this list is maintained in the outputted name
+        Accepted Included Names include: "direction", "latitude", "longitude", "date_str", "lc_id"
+    additional_name_stem : str, default=""
+        Additional custom information the user can add to the name.
```

Comment:
See additional name comment in the tests

Reply:
see https://github.com/IGES-Geospatial/globe-observer-utils/pull/73#discussion_r776763168